examples/Python Tooling Scripts/params_example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "SourceAccountAPIKey": "original_account_api_key",
+  "APIUrl": "api_region_endpoint",
+  "TargetAccountAPIKEy": "copy_to_account_api_key"
```

Comment:
Suspect this capital E in Key is not desired

Reply:
I honestly didnt do anything with the examples from the [Automation repo](https://github.com/itglue/automation). I simply migrated them into this repos new 'examples' folder.

Either way, that's an easy fix

Reply:
Sounds good. Thanks